src/words/fast.jl

```diff
@@ -175,9 +175,9 @@ can be used for splitting tokens.
 For example, `cannot` would be split into `can` and `not`.
 `freeeee` would be replaced with `free`
 """
-function replaces(ts, ss)
+function replaces(ts, ss, boundary = true)
```

Comment:
This was done to enable using rules `LSTRIP` and `RSTRIP` in `toktok_tokenize` while reusing the code.

Reply:
Seems reasonable, needs to be added to docstring